wing-redis/utils.js

```diff
@@ -0,0 +1,54 @@
+const child_process = require("child_process");
+const IoRedis = require("ioredis");
+
+
+exports.shell = async function (command, args, cwd) {
+    return new Promise((resolve, reject) => {
+        child_process.execFile(command, args, { cwd }, (error, stdout, stderr) => {
+            if (error) {
+                console.error(stderr);
+                return reject(error);
+            }
+
+            return resolve(stdout ? stdout : stderr);
+        });
+    });
+};
+
```

Comment:
I would go with a different approach:

```js
interface IRedisClient {
  inflight get(key: str): str;
  inflight set(key: str, ...): void;
  // ....
}

class Util {
  extern "./redis.js"
  static inflight newClient(url): IRedisClient;
}
```

And then, in redis.js:

```js
const IoRedis = require("ioredis");

exports.newClient = (url) => new IoRedis(url);
```

And that's it. Basically then you can just use the client from wing.